githook/prepare-commit-msg

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+
+import sys
+
+def is_integer(value: str) -> bool:
+    try:
+        int(value)
+        return True
+    except ValueError as _:
+        pass
+
+    return False
+
+def check_commit_message() -> int:
+    commit_message_filename: str = sys.argv[1]
+    with open(commit_message_filename) as file:
+        first_line: str = file.readline().strip()
+
+    issue_name = first_line.split(" ")[0]
+
+    # Allow fixup commits
+    if issue_name == "fixup!":
+        return 0
+
+    if issue_name == "WIP":
+        return 0
+
+    if issue_name.startswith("KTOR-") and is_integer(issue_name[5:]):
+        return 0
+
+    print(f"Commit name should starts with issue number(like KTOR-1234). Current prefix is '{issue_name}'")
+    return 1
```

Comment:
It's not obvious what the return value means.

Reply:
It's an exit code. Let me introduce named constants

Reply:
I know what it's. You should have a function that checks the presence of an issue in the commit message and returns boolean.

Reply:
Yep, let me refactor that